types/streamify-string/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for streamify-string 1.0
+// Project: https://github.com/chrisallenlane/streamify-string
+// Definitions by: Jesse Wright <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Readable } from 'stream';
+
+/**
+ * Converts a string into a Node readable stream.
+ */
+declare function streamifyString(str: string, options?: any): Readable;
```

Comment:
from looking at the source code for this package, maybe this function should actually be a class?
in the source, we have `module.exports = Streamify`, and `function Streamify(...) { ... }` is a constructor. so my suggestion here would be to have something like this instead:
```ts
declare class Streamify extends Readable {
    constructor(str: string, options?: any);
}

export = Streamify;
```

Reply:
The problem that I am running into with this suggestion is that it requires the `new` keyword to be used since `Streamify` is now a class; but the following js runs without error

```js
import streamifyString from 'streamify-string';

const stream = streamifyString('data');
stream.on('data', (d) => console.log('Data: ' + d));
stream.on('end', () => console.log('Done!'));
```

Reply:
oh, you should add interface + class, so one can use `new` + plain `Func` creator.
But it still is a class like (see notation in readme):
```ts
// would require allow empty interface exclusion
interface Streamify extends Readable {}
interface StreamifyConstructor {
  new (str: string, options?: {}): Streamify;
  (str: string, options?: {}): Streamify;
}
declare const Streamify: StreamifyConstructor;
```

Reply:
Makes sense - I really appreciate the feedback :)